signer/src/proto/convert.rs

```diff
@@ -379,6 +1669,13 @@ mod tests {
         }
     }
 
+    impl Dummy<Unit> for Point {
+        fn dummy_with_rng<R: rand::Rng + ?Sized>(_: &Unit, rng: &mut R) -> Self {
+            let public_key: PublicKey = Faker.fake_with_rng(rng);
+            public_key.into()
```

Comment:
I would change this to be something like
```rust
let number: [u8; 32] = Faker.fake_with_rng(rng);
let scalar = p256k1::scalar::Scalar::from(number);
Point::from(scalar)
```
This approach catches the case where we would generate a point that is not a public key (even though we will never trigger it by chance).